Week1/SantiagoMedina/Collections/Map/src/test/java/mapComparison/AppTest.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package mapComparison;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class AppTest {
+    @Test public void testAppHasAGreeting() {
```

Comment:
Do you need this empty test for some reason? if not, remove it

Reply:
you are right, it is not neccesary. i fixed.